### PR TITLE
test reviews

### PR DESCRIPTION
**Pull request by ifeify**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/ifeify/ReviewNinja-Welcome/pull/1" target="_blank">ReviewNinja

File: ReadMe.md [hunk 1]
```diff
@@ -1 +1,2 @@
-# ReviewNinja-Welcome
\ No newline at end of file
+# ReviewNinja-Welcome
```
Comment (ifeify) [inline ReadMe.md]: !completed

File: ReadMe.md [hunk 2]
```diff
@@ -1 +1,2 @@
-# ReviewNinja-Welcome
\ No newline at end of file
```
Comment (ifeify) [inline ReadMe.md]: This is essentially the same thing

Comment (ifeify) [inline ReadMe.md]: !fix

Comment (ifeify) [inline ReadMe.md]: !fixed

**Pull request merged**